benchmarks/apps/sombrero/sombrero.py

```diff
@@ -122,4 +137,8 @@ def set_up_from_parameters(self):
 
     @run_after('setup')
     def setup_num_tasks(self):
-        self.num_tasks = self.current_partition.processor.num_cores * 64
+        self.num_tasks_per_node = max_num_tasks(
+            self.current_partition.processor.num_cores,
+            LATTICE_VOLUME_MEDIUM // 64,
+        )
+        self.num_tasks = self.num_tasks_per_node * 64
```

Comment:
One very tiny comment. Could you replace the two uses of the magic number `64` with a variable e.g. `num_nodes`?

Reply:
Done!  Can you please check it works for you as expected?